dashboards/datadog_audit_trail_rate_limits_tracker/README.md

```diff
@@ -0,0 +1,27 @@
+# Audit Trail : Rate Limits Tracker
+
+Overview of Datadog rate limits being experienced by end-users
```

Comment:
Can we add some more information. Maybe links to documentation around API rate limits?